apps/condo/migrations/20210802144240-0033_manual_localize_default_organization_employee_roles.js

```diff
@@ -0,0 +1,43 @@
+exports.up = async (knex) => {
+    // maybe read deafults.json and iterate over it here
+    // to dynamically replace localzied names?
+
+    // is there need to change fingerprint and down this mutation?
+    await knex.raw(`
+    BEGIN;
+        -- Administrator
+        UPDATE "OrganizationEmployeeRole"
+        SET name = 'employee.role.Administrator.name'
+        WHERE (name = 'Administrator' OR name = 'Администратор') AND "createdAt" = "updatedAt" AND "createdBy" = "updatedBy";
```

Comment:
why you need `AND "createdAt" = "updatedAt" AND "createdBy" = "updatedBy"` ?

Reply:
@pahaz to be sure it was not changed